src/Search.tsx

```diff
@@ -18,6 +19,23 @@ export default function Search({ updateSearch, search }: SearchProps) {
   const ref = React.useRef<HTMLInputElement>(null);
   const iconSize = 24;
   const inputHeight = 36;
+
+  // Select contents of search input when switching to the page
+  React.useEffect(() => {
+    ref?.current?.select();
+  }, [ref]);
+
+  // Select contents of search input when pressing cmd/ctrl-f
+  useShortcut(
+    (event: KeyboardEvent): void => {
```

Comment:
Can be simplified to `event => {` because the type signature is inferred based on the definition of `useShortcut`